hardware/Drivetrain.java

```diff
@@ -22,7 +22,7 @@
 
     private boolean slow_mode = false;
 
-    private static final double     COUNTS_PER_MOTOR_REV    = 1120;
+    private static final double     COUNTS_PER_MOTOR_REV    = 537.6;
```

Comment:
lets test this today to see if motors go slower or faster; it may result in changes in teledrive slowmode